cmd/secret/cmd/root_test.go

```diff
@@ -216,15 +295,57 @@ func TestRoot_Server(t *testing.T) {
 			require.EqualValues(t, http.StatusOK, resp.StatusCode)
 			require.NoError(t, resp.Body.Close())
 		})
-		t.Run("middleware check success", func(t *testing.T) {
+		t.Run("expect redis get method success", func(t *testing.T) {
+			expectedSipherKey := "key value"
```

Comment:
Add one more test case that checks that when `method="local"` used for `set` then the key could not be fetched by `method="remote"` and vise-versa.